src/data_extractor/data_getter.py

```diff
@@ -0,0 +1,50 @@
+import os
+import csv
+
+
+class DataGetter:
+    CURRENT_DIR = os.path.dirname(os.path.realpath(__file__))
+    TOWN_DATA = CURRENT_DIR + "/../../towns.txt"
+    CSV_TRAIN_DATA = CURRENT_DIR + "/../../data/Test_rev1.csv"
+
+    @classmethod
+    def get_towns(cls):
+        """
+        get list of towns above 50000
+        :return:list of towns
+        """
+        with open(cls.TOWN_DATA, 'r') as f:
+            cities = []
+            for city in f.readlines():
+                cities.append(city.lower().replace("\n", ""))
+            return cities
+
+    @classmethod
+    def get_raw_training_data(cls, header=None):
```

Comment:
Just curious, any reason why not read data with pandas in a single line ? pd.read_csv(file)